frontend/src/components/Filters/Filters.jsx

```diff
@@ -0,0 +1,66 @@
+import Checkbox from '../Checkbox/Checkbox'
+import InputFilters from '../InputFilters/InputFilters'
+import './Filters.scss'
+
+function Filters({ stateVisible }) {
+  return (
+    <div>
+      {stateVisible === true ? (
+        <div className="filters">
+          <div className="filters__dates">
+            <div className="filters__registration">
+              <p className="filters__title text-m-bold">Дата оформления</p>
+              <div className="filters__inputs">
+                <InputFilters placeholder={'c'} />
```

Comment:
Необязательно использовать скобки при передаче строки
```suggestion
                <InputFilters placeholder="c" />
```